static-analysis/assert-alpha.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Webmozart\Assert\StaticAnalysis\AssertString;
+
+use Webmozart\Assert\Assert;
+
+/**
+ * @psalm-param string|numeric $value
+ */
+function consume($value) : string
```

Comment:
What exactly is the purpose of these consume functions?

Are they intended to be called by the users? Or is it just some static analysis magic?

Reply:
As discussed in person, these are just test files to verify that the inferred types pass the function return type declaration type checks.